VTEX - Marketplace Protocol - External Marketplace Mapper.json

```diff
@@ -405,335 +592,252 @@
+                                            "description": "Indicates whether connector is active.",
+                                            "type": "boolean"
                                         },
                                         "isCertified": {
-                                            "type": "boolean",
-                                            "title": "isCertified",
-                                            "example": false
+                                            "description": "Indicates whether the connector is certified.",
+                                            "type": "boolean"
                                         },
                                         "contactInfo": {
+                                            "description": "Seller contact information.",
                                             "type": "object",
-                                            "title": "contactInfo",
-                                            "required": [
-                                                "name",
-                                                "email",
-                                                "phone",
-                                                "defaultMessage"
-                                            ],
                                             "properties": {
                                                 "name": {
-                                                    "type": "string",
-                                                    "title": "name",
-                                                    "example": "Jhon"
+                                                    "description": "Name of the person responsible for the marketplace account interested in connecting to the merchant.",
+                                                    "type": "string"
                                                 },
                                                 "email": {
-                                                    "type": "string",
-                                                    "title": "email",
-                                                    "example": "jhon@marketplace.com"
+                                                    "description": "Email address of the person responsible for the marketplace account interested in connecting to the merchant.",
+                                                    "type": "string"
                                                 },
                                                 "phone": {
-                                                    "type": "string",
-                                                    "title": "phone",
-                                                    "example": "(77) 7777-7777"
+                                                    "description": "Phone of the person responsible for the marketplace account interested in connecting to the merchant.",
```

Comment:
Especialmente neste contexto de uso de conector, seria bom utilizar outra expressão que não "connecting to the merchant". Não se conecta ao merchant propriamente.

Reply:
Que outro termo você usaria? Com o que o marketplace se conecta?

Reply:
Pode ser "merchant's account", desde que não seja só "merchant".

Reply:
Do [glossário de localization](https://docs.google.com/spreadsheets/d/1vSSg5CrimqSxvWjBoXLE5P7ATJCdRnPT/edit?gid=1978438735#gid=1978438735)
> A merchant is a person or business that accepts payments from customers and clients for goods or services. The main user of the VTEX platform.

Reply:
Boa, então mantém.